bindings/nodejs/lib/types/models/info/node-info-protocol.ts

```diff
@@ -102,6 +102,46 @@ export interface ProtocolParameters {
      * The parameters used by signaling protocol parameters upgrade.
      */
     versionSignaling: VersionSignalingParameters;
+    /**
+     * Rewards Parameters defines the parameters that are used to calculate Mana rewards.
+     */
+    rewardsParameters: RewardsParameters;
+}
+
+/**
+ * Rewards Parameters defines the parameters that are used to calculate Mana rewards.
+ */
+export interface RewardsParameters {
+    /**
+     * The number of validation blocks that should be issued by a selected validator
+     * per slot during its epoch duties.
+     */
+    validationBlocksPerSlot: number;
+    /**
+     * Profit Margin Exponent is used for shift operation for calculation of profit margin.
+     */
+    profitMarginExponent: number;
+    /**
+     * The length in epochs of the bootstrapping phase.
+     */
+    bootstrappingDuration: number;
```

Comment:
```suggestion
    bootstrappingDuration: u64;
```

Reply:
this is length in epochs, so will be u32 soon right? Thats why i left it

Reply:
Ah yeah probably will also be an u32 then